### DISCUSSION TITLE
Problem with ambient occlusion in AR mode

### DISCUSSION THREAD

**Discussion by clydepashley** [google/model-viewer#4392] [Q&A]
Hi, in my models, I have an AO texture using a second UV map. Everything displays fine in non-ar mode. When I select the 'view in your space' option, it seems to show the AO wrongly. Any idea why?

![IMG_0924](https://github.com/google/model-viewer/assets/5012515/e97d2f65-ce79-44ba-84be-b8e80e7ade14)

**Comment by elalish** [comment]
By AR mode, I think you specifically mean iOS, which implies Quick Look. This looks like a USDZ autogeneration problem, which isn't too surprising when using advanced features like this. I would recommend filing a bug on three.js' USDZExporter. Meanwhile, you can generate your own USDZ with Apple's tools and supply that as an `ios-src`.
